File: src/main/java/com/randomperson22/piggymodtest/entities/teacher/EntityTeacher.java
package com.randomperson22.piggymodtest.entities.teacher;

import com.randomperson22.piggymodtest.entities.EntityPiggyBase;
import com.randomperson22.piggymodtest.init.ModSounds;
import net.minecraft.util.SoundEvent;
import net.minecraft.world.World;

public class EntityTeacher extends EntityPiggyBase {
    public EntityTeacher(World worldIn) {
        super(worldIn);
    }

    @Override
    public SoundEvent getJumpscareSound() {
        return ModSounds.ENTITY_TEACHER_JUMPSCARE;
    }
}

File: src/main/kotlin/com/randomperson22/piggymodtest/entities/teacher/EntityTeacher.kt
package com.randomperson22.piggymodtest.entities.teacher

import com.randomperson22.piggymodtest.entities.EntityPiggyBase
import com.randomperson22.piggymodtest.init.ModSounds
import net.minecraft.util.SoundEvent
import net.minecraft.world.World

class EntityTeacher(worldIn: World) : EntityPiggyBase(worldIn) {
    override val jumpscareSound: SoundEvent
        get() = ModSounds.ENTITY_TEACHER_JUMPSCARE
}